server/lobbyconnection.py

```diff
@@ -287,8 +287,8 @@ async def send_updated_achievements(self, updated_achievements):
     async def command_admin(self, message):
         action = message['action']
 
-        if self.player.admin:
-            if action == "closeFA":
+        if action == "closeFA":
```

Comment:
I'm not going to make the actions constants for now. We may find a better solution for this when we work on refactoring the protocol for auto documentation. But I'm not going to do it in this PR.

As for the permissions. I'm not really sure. Seems a bit overkill since they're only used in this one function.